.changeset/sweet-hotels-cross.md

```diff
@@ -4,15 +4,15 @@
 
 Adds the experimental support for the [static headers Astro feature](https://docs.astro.build/en/reference/adapter-reference/#experimentalstaticheaders).
 
-When the feature is enabled via option `experimentalStaticHeaders`, and CSP is enabled, the integration will the CSP headers to the `config.json` file.
+When the feature is enabled via option `experimentalUseStaticHeaders`, and CSP is enabled, the integration will the CSP headers to the `config.json` file.
```

Comment:
Sorry, I just meant the local names of the vars! I think `experimentalStaticHeaders` is fine for the public option